qemu/tests/cfg/win_virtio_driver_update_test.cfg

```diff
@@ -87,9 +90,27 @@
             iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -i 0 -i 1 -b iozone.xls -f ${disk_letter}:\testfile"
             iozone_timeout = 7200
             wait_bg_time = 180
+        - with_viorng:
+            no no_virtio_rng
+            driver_name = viorng
+            device_name = "VirtIO RNG Device"
+            run_bgstress = rng_bat 
+            target_process = random\w*.exe
+            session_cmd_timeout = 240
+            read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
```

Comment:
Hi @suqinhuang 

Same problem of rng for windows guest.

2016-08-18 14:07:13: C:\Users\Administrator>
2016-08-18 14:07:13: D:\random_%PROCESSOR_ARCHITECTURE%.exe
2016-08-18 14:07:13: Generated random number is
2016-08-18 14:07:13: 0x7f, 0x97, 0x0d, 0x42, 0x57, 0x61, 0x72, 0xd1, 0x00, 0x8f,
2016-08-18 14:07:13: 0x83, 0x42, 0xf5, 0x16, 0x10, 0x0b, 0x21, 0x6c, 0x82, 0x43,
2016-08-18 14:07:13: 0xe3, 0x8a, 0x1e, 0x7f, 0x9e, 0xd9, 0xfb, 0x8d, 0x1a, 0x37,
2016-08-18 14:07:13: 0x77, 0x82, 0x77, 0x55, 0x9c, 0x1c, 0x11, 0xe7, 0x21, 0x0a,
2016-08-18 14:07:13: 0x7e, 0xa5, 0xe8, 0x82, 0x13, 0xc8, 0x77, 0x86, 0x2f, 0x41,
2016-08-18 14:07:13: 0xb2, 0x63, 0x8b, 0x1d, 0x51, 0x33, 0x44, 0xcb, 0xdb, 0x4c,
2016-08-18 14:07:13: 0xbb, 0x8e, 0x01, 0x8d, 0xf4, 0xdc, 0x13, 0x80, 0x79, 0xc7,
2016-08-18 14:07:13: 0x10, 0x03, 0x7b, 0xc5, 0xba, 0x0d, 0x53, 0x32, 0x39, 0x1b,
2016-08-18 14:07:13: 0x98, 0x5c, 0xd3, 0xef, 0xcd, 0xa7, 0xf3, 0xfa, 0xc0, 0x15,
2016-08-18 14:07:13: 0x85, 0x93, 0x40, 0x88, 0xcb, 0x78, 0x8e, 0x0b, 0x9c, 0x52,
2016-08-18 14:07:13: 0xf4, 0x09, 0x8d, 0xff, 0x1e, 0x82, 0xbe, 0x0f, 0x06, 0xc4,
2016-08-18 14:07:13: 0x38, 0x4a, 0x48, 0x3b, 0x13, 0x0e, 0x7c, 0xcb, 0xc6, 0xa0,
2016-08-18 14:07:13: 0x86, 0x0c, 0xf9, 0xe9, 0x2a, 0xa0, 0xcd, 0x0b,

Time if too short to check the rng generation process for Win2012r2 guest.
14:09:17 ERROR| FAIL 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.rng_random.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.win_virtio_driver_update_test.during_bg_test.uninstall_install.with_viorng -> TestFail: Backgroud test rng_bat is not alive!